gemelli/q2/tests/simulations.py

```diff
@@ -0,0 +1,244 @@
+from __future__ import division
+# utils
+import pandas as pd
+import numpy as np
+# blocks
+from scipy.stats import norm
+from numpy.random import poisson, lognormal
+from skbio.stats.composition import closure
+# Set random state
+rand = np.random.RandomState(42)
+
+
+def Homoscedastic(X_noise, intensity):
```

Comment:
suggested name change:
`Homoscedastic` -> `make_homoscedastic`, `Heteroscedastic` -> `make heteroscedastic`.
Kind of inline with `sklearn`'s `make_classification`, etc. https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_classification.html

Reply:
Cancel that. (after reading rest of code in this module)